.rubocop/style.yml

```diff
@@ -1,4 +1,7 @@
 ---
+Style/ArrayIntersect:
```

Comment:
This doesn't seem like a bad idea https://docs.rubocop.org/rubocop/cops_style.html#stylearrayintersect, but maybe there are a lot of hits.
Maybe running `./bin/rubocop --regen-todo` for these, since there likely needs to be a change to that file for the new Rubocop version in the header

Reply:
Regen'd todo.